services/quiz.js

```diff
@@ -109,14 +117,26 @@ function generateConjugation(verb, pronoun, moodTense) {
 	return null;
 }
 
-function generateQuestion() {
+function generateQuestion(verbSet) {
 	const moodTense = chooseMoodTense();
 	const pronoun = choosePronoun();
-	const verb = chooseVerb();
+	const data = verbSets[verbSet];
+	let verbs=[];
+	let hasDefinition = false;
+	if (data.constructor === Array){
+		verbs = data;
+	}
+	else {
+		verbs = Object.keys(data);
+		hasDefinition = true;
+	}
+
+	const verb = chooseVerb(verbs);
 	const reflexive = isReflexive(verb);
 	const question = {
 		pronoun: pronoun.name,
 		verb: verb,
+		definition: hasDefinition ? data[verb] : '',
```

Comment:
@sdee you can get rid of hasDefinition and just use data[verb] here. If it is defined use it, if not use empty string.